freqtrade/freqtradebot.py

```diff
@@ -938,13 +960,13 @@ def _safe_sell_amount(self, pair: str, amount: float) -> float:
             raise DependencyException(
                 f"Not enough amount to sell. Trade-amount: {amount}, Wallet: {wallet_amount}")
 
-    def execute_sell(self, trade: Trade, limit: float, sell_reason: SellType) -> None:
+    def execute_sell(self, trade: Trade, limit: float, sell_reason: SellType) -> bool:
```

Comment:
I guess we were thinking of opposite: make `execute_buy()` returning None...
Anyway, I understand your possible rationaly here (to have it aligned with `execute_buy()`), so it's okay, this can be reworked later...

Reply:
It's also a way to communicate the status of the execute buy/sell.